test/components/SplashScreen/SplashScreen.test.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import { render, screen } from '@testing-library/react';
+
+import { SplashScreen } from '../../../app/components';
+
+describe('SplashScreen', () => {
+  test('renders to screen when called', () => {
+    render(<SplashScreen />);
+
+    expect(screen.queryByTestId('splash-screen')).not.toBeNull();
```

Comment:
I think there is some nuance between `.toBeNull` and `.toBeInTheDocument`. The former checks if the component exists, and the latter checks if the component is rendered at the moment of the test. I think you want `.toBeInTheDocument` here.